src/main.ts

```diff
@@ -1,4 +1,6 @@
+// @ts-expect-error - no types found
```

Comment:
Why are these needed

Reply:
Sometimes my IDE complains about the types then other times it doesn't and I can't stand red file names but I'll remove. Never happen to you on Windows, I assumed that's what it was lmao.